distribution/lib/Standard/Snowflake/0.0.0-dev/src/Internal/Snowflake_Dialect.enso

```diff
@@ -0,0 +1,12 @@
+from Standard.Base import all
```

Comment:
Can this module be marked `private`?

Reply:
yes will make the dialect private when I restructure it in next PR.